Formula/prettyping.rb

```diff
@@ -6,6 +6,9 @@ class Prettyping < Formula
 
   bottle :unneeded
 
+  # Fix IPv6 handling on BSD/OSX: https://github.com/denilsonsa/prettyping/pull/11
+  patch :DATA
```

Comment:
I see, that's a shame. Would you mind submitting the patch here to https://github.com/Homebrew/formula-patches then and reworking it like [this](https://github.com/Homebrew/homebrew-core/blob/7f5799fd2178001b715827e3a84e021923d357c2/Formula/cdparanoia.rb#L27-L30)? We try to avoid inline patches because people find novel ways to break them.